keyboards/nullbitsco/snap/config.h

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2021 Jay Greco
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+// clang-format off
+#pragma once
+
+#include "config_common.h"
+
+/* Used to set host for remote KB if VUSB detect doesn't work. */
+// #define KEYBOARD_HOST // Force host mode
+// #define KEYBOARD_REMOTE // Force remote mode
+
+// Workarounds for boot/sleep issues
+#define USB_SUSPEND_WAKEUP_DELAY 250
+#define NO_SUSPEND_POWER_DOWN
+
+/* space savers */
+// #define NO_ACTION_TAPPING // Waiting on QMK #11528
+#define NO_ACTION_ONESHOT
+#define IGNORE_MOD_TAP_INTERRUPT
+#define TAPPING_FORCE_HOLD
```

Comment:
These should be defined at the keymap level, not here.

Reply:
I missed this on the last update -- fixed now!